src/sentry/templates/sentry/403-csrf-failure.html

```diff
@@ -33,5 +42,93 @@ <h2>{% trans "CSRF Verification Failed" %}</h2>
         {% endif %}
 
         <p>Read more about <a href="http://en.wikipedia.org/wiki/Cross-site_request_forgery">CSRF on Wikipedia</a>.</p>
+
+        <script>
+          const status = document.getElementById('csrf-status');
+          const button = document.getElementById('csrf-rotate-btn');
+
+          /**
+           * Automatically refresh page to prevent form resubmission
+           */
+          function forceRefreshPage() {
+            setTimeout(() => {
+                window.location.replace(window.location.href)
```

Comment:
if you want analytics, can you put a `referrer` on the refreshed page? I think we automatically have analytics if you put the `referrer` param

Reply:
`window.location.reload(true)` doesn't work on Chrome 😢